hardhat.config.js

```diff
@@ -41,7 +42,7 @@ module.exports = {
         }
     },
     etherscan: {
-        apiKey: process.env.ETHERSCAN_APIKEY
+        apiKey: process.env.ETHERSCAN_API_KEY
```

Comment:
Why is this being changed?
It's causing CI to fail.
```suggestion
        apiKey: process.env.ETHERSCAN_API_KEY
```